### DISCUSSION TITLE
Postgres function (RPC) help

### DISCUSSION THREAD

**Discussion by Steccah** [supabase/supabase#21118] [Questions] [answered]
Relation upvote not found even though i wrote Upvote, am i doing something wrong?

![image](https://github.com/supabase/supabase/assets/21223192/abd50f63-2137-4dd6-a80f-4ff472991f6c)

**Comment by GaryAustin1** [comment, marked as answer]
You should not use capital letters in Postgres for names of tables, columns, functions.  It will cause you grief.
Anytime you use capital letters in names you have to use double quotes around the name as "Upvote".
